projection/core/src/multi-jvm/scala/com/lightbend/lagom/internal/projection/ProjectionRegistrySpec.scala

```diff
@@ -172,28 +172,37 @@ class ProjectionRegistrySpec
 
       val testProbe = registerProjection(projectionName, tagNames.toSet)
       testProbe.ignoreMsg {
-        case _ => true
+        case FakeStarting(`tagName001`) => false
+        case _                          => true
```

Comment:
Ignore all message expect for the one coming from tagName001

Reply:
@ignasi35, I'm keeping this ignore, but checking that the other tags have the expected status at least. 

It's hard to observe all message in all nodes because they come out of order and on different nodes. Tried with `expectMsgAllOf` without success. Until I realised that it would never work because it's multi-node nature and the `MultiNodeTest` introduced here doesn't support it.

At least, with the `ignoreMsg`, we can observe that the projection that we are stopping and restarting are emitting the expected messages.